src/script/interpreter.cpp

```diff
@@ -1248,6 +1248,7 @@ uint256 SignatureHash(
                     throw std::logic_error(
                         "ZIP 244: Used SIGHASH_SINGLE without a corresponding output");
                 }
+                [[fallthrough]];
```

Comment:
```suggestion
                break;
```
(Equivalent in this case.)